index.html

```diff
@@ -2035,10 +2035,30 @@
 
       <section>
         <h3>Bundling Dependent Claims</h3>
+        <p>
+It is considered a best practice for issuers to atomize information in a
+credential, or use a signature scheme that allows for selective disclosure.
+In the former case, if the atomization is not done securely by the issuer,
+the holder may bundle together different credentials in a way that was not
```

Comment:
Suggest 'may' -> 'might' or 'might attempt to' to avoid confusion with normative MAY.